examples/two-columns-layout/src/plugins/popularCategoriesPlugin.tsx

```diff
@@ -0,0 +1,92 @@
+/** @jsx h */
+import {
+  AutocompleteComponents,
+  AutocompletePlugin,
+  getAlgoliaFacets,
+} from '@algolia/autocomplete-js';
+import { h } from 'preact';
+
+import { ALGOLIA_PRODUCTS_INDEX_NAME } from '../constants';
+import { searchClient } from '../searchClient';
+import { PopularCategoryHit } from '../types';
+
+const images = {
+  Women:
+    'https://res.cloudinary.com/hilnmyskv/image/upload/v1646067858/women_category_vwzkln.jpg',
```

Comment:
The `"https://res.cloudinary.com/hilnmyskv/image/upload/"` can probably be stored in a variable and reused, this way Prettier won't do two-line formatting.